CHANGES.rst

```diff
@@ -6,6 +6,11 @@ PRAW follows `semantic versioning <http://semver.org/>`_.
 Unreleased
 ----------
 
+**Added**
+
+- :meth:`.Inbox.mark_all_read` to mark all messages as read with one API call.
+- :meth:`.InboxableMixin.unblock_subreddit` to unblock a subreddit.
```

Comment:
```suggestion
- :meth:`.unblock_subreddit` to unblock a subreddit.
```
Could we make this `` :meth:`.unblock_subreddit` `` or `` :meth:`~.InboxableMixin.unblock` `` if we make it work for non-subreddit objects?

Reply:
Do you still want me to change these to 

     :meth:`.mark_all_read`
     :meth:`.unblock_subreddit` 

? 
Or perhaps

     :meth:`~.Inbox.mark_all_read`
     :meth:`~.InboxableMixin.unblock_subreddit` 

Is the policy to only use the class when there is a conflict? Sometimes this will require you to change old changelog entries when another method is added with the same name.

Reply:
Usually, the shorter the better unless the method name is used in multiple classes. In those instances, if it still renders in a way that the user could easily understand how the method is used then leaving the more qualified name visible would be fine. 

For like: `` :meth:`.Inbox.read_all_messages` `` I'm fine with it either way because it's still kinda clear that the method would be used like so: `reddit.inbox.read_all_messages()`. Now `` :meth:`.InboxableMixin.unblock_subreddit` `` on the other hand, it could be confusing as to where to find the method since `InboxableMixin` is not a class one would utilize directly. So it would be better to have it just as `` :meth:`.unblock_subreddit` `` or even `` :meth:`~.InboxableMixin.unblock_subreddit` `` as both would render the same way. I would prefer the former since the it would be redundant to have it more qualified as there is currently only one `unblock_subreddit` method.

Sphinx can resolve by just the method name unless there is more than one method with the same name. In that case, it will need to be more qualified. Also, adding the `~` will shorten the more qualified reference to just the method name.

Side notes: 

I currently have a PR open to make the reference links and inline literals more consistent throughout the code base but I haven't had the time to dedicate to it.

I also wanted to apologize for some of the nitpicky like suggestions. I'm very detail oriented and when I see an inconsistency I want to go through everything and make it consistent and I'm just hoping you're not taking my nitpickyness the wrong way.

Also, come join our [slack](https://join.slack.com/t/praw/shared_invite/enQtOTUwMDcxOTQ0NzY5LWVkMGQ3ZDk5YmQ5MDEwYTZmMmJkMTJkNjBkNTY3OTU0Y2E2NGRlY2ZhZTAzMWZmMWRiMTMwYjdjODkxOGYyZjY) if you'd like to.😛

Reply:
I don't mind suggestions. I should have caught the sorting issues before I pushed. I have multiple versions of endpoints.py and docs/code_overview/other.rst, which led to some mistakes slipping in.

I also modified a CHANGES.rst entry that I made in an earlier commit. There were three items that I meant to quote, not italicize.

I'll try to check out Slack sometime soon.